board/linux/crashdump-serial-number/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+add_library(crashdump-serial-number STATIC)
+
+target_include_directories(crashdump-serial-number
+    PUBLIC
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_LIST_DIR}>/include
+)
+
+target_sources(crashdump-serial-number
+    PUBLIC
+        crashdump_serial_number.cpp
+        include/crashdump-serial-number/crashdump_serial_number.hpp
```

Comment:
It seems to me that no header should be mentioned explicitly as a _source file_, at least in case of a usual library as we have here.